src/main/java/io/openliberty/tools/intellij/actions/LibertyGeneralAction.java

```diff
@@ -12,8 +21,11 @@
 import io.openliberty.tools.intellij.LibertyPluginIcons;
 import io.openliberty.tools.intellij.util.Constants;
 import io.openliberty.tools.intellij.util.LibertyProjectUtil;
+import io.openliberty.tools.intellij.util.LocalizedResourceUtil;
 import org.jetbrains.annotations.NotNull;
 
+import java.text.MessageFormat;
```

Comment:
Is this import needed here and 5 other files or is it left over from intermediate work?